drivers/base/bootvid/common.c

```diff
@@ -319,6 +387,88 @@ VidSetScrollRegion(
     VidpCurrentY = Top;
 }
 
+/*
+ * @implemented
+ */
+VOID
+NTAPI
+VidDisplayString(IN PUCHAR String)
+{
+    ULONG TopDelta = BOOTCHAR_HEIGHT + 1;
```

Comment:
I think we can remove this variable altogether and replace all the "TopDelta" occurrences by its value (which is fixed at compile-time by the way).